addons/product/tests/test_product_attribute_value_config.py

```diff
@@ -408,11 +405,27 @@ def test_get_first_possible_combination(self):
             'value_ids': [(6, 0, [color_red.id, color_green.id])],
         })
 
+        computer2 = self.env['product.template'].create({'name': 'Computer2'})
```

Comment:
the computer is no longer deleted thanks to the change, you might not need `computer2` anymore